pikaur/print_department.py

```diff
@@ -128,7 +129,7 @@ def pretty_format_upgradeable(  # pylint: disable=too-many-statements
 
     SortKey = Union[Tuple, str]
 
-    def pretty_format(pkg_update: 'InstallInfo') -> Tuple[str, SortKey]:  # pylint:disable=too-many-locals
+    def pretty_format(pkg_update: 'InstallInfo') -> Tuple[str, SortKey]:  # pylint:disable=too-many-locals,R0912
```

Comment:
and please use error-name instead of error-code

sorry for nitpicking :D

Reply:
i actually tried to use the error-name, but then i got a line length warning so figured it would be better if i just used the code. still want it to be changed?

Reply:
ok let's leave it as is for now